.github/CODEOWNERS

```diff
@@ -1,13 +1,13 @@
 #cpp code owners
-cpp/               @rapidsai/<repo>-cpp-codeowners
+cpp/               @rapidsai/cucim-cpp-codeowners
 
 #python code owners
-python/            @rapidsai/<repo>-python-codeowners
-python/dask_cudf/  @rapidsai/<repo>-dask-codeowners
+python/            @rapidsai/cucim-python-codeowners
```

Comment:
Would it be possible to add @quasiben, @grlee77, and myself to this team?

Reply:
done: https://github.com/orgs/rapidsai/teams/cucim-python-codeowners